### PR TITLE
Update dbt integration doc with new open-source and Cloud sections

### PR DESCRIPTION
**Pull request by williebsweet**

### CODE REVIEW

File: docs/os_diff/dbt_integration.md [hunk 1]
```diff
@@ -1,144 +1,157 @@
...
+  ```
+  To learn more about setting primary key tags, check out [this section](/integrations/orchestration/dbt_adv_config#primary-keys).
+
+### Run with --dbt
+
+Run your dbt model with `data-diff --dbt` to see the impact that your model change had on the data.
     
+  ```zsh
+  dbt run --select <MODEL> && data-diff --dbt
+  ```
```
Comment (dlawin) [inline docs/os_diff/dbt_integration.md:106]: should we call out that it can also be ran as

```zsh
dbt run --select <MODEL>
data-diff --dbt
```

Comment (williebsweet) [reply docs/os_diff/dbt_integration.md:106]: Done - https://github.com/datafold/datafold-docs/pull/136/commits/055aeec022fc2fd08f8c6b70bc67fe53ae5c24ae

File: docs/os_diff/dbt_integration.md [hunk 2]
```diff
@@ -1,144 +1,174 @@
...
+  
+  <details>
+    <summary>Optional arguments</summary>
+    scenario_1: <br/> <code>placeholder</code> <br/>
+    <br/>
+    scenario_2: <br/> <code>placeholder</code> <br/>
+    <br/>
+    scenario_3: <br/> <code>placeholder</code> <br/>
+    <br/>
+  </details>
```
Comment (williebsweet) [inline docs/os_diff/dbt_integration.md:98]: @dlawin can you document the 3 scenarios and configurations here?

File: docs/os_diff/dbt_integration.md [hunk 3]
```diff
@@ -1,144 +1,194 @@
...
+    
+  ```bash
+  # as one command
+  dbt run --select <MODEL> && data-diff --dbt
+  ```
+  ```bash
+  # or as separate commands
+  dbt run --select <MODEL>
+  data-diff --dbt
+  ```
```
Comment (dlawin) [inline docs/os_diff/dbt_integration.md:194]: can we do this above in the "Run with --dbt" section as well?

File: docs/os_diff/dbt_integration.md [hunk 4]
```diff
@@ -1,144 +1,194 @@
...
-```
+### Run with --cloud
 
-<center><img src={ValuesTab} style={{width: '75%'}}/></center>
\ No newline at end of file
+Run your dbt model with `--cloud` to see the impact that your model change had on the data.
+    
+  ```bash
+  # as one command
+  dbt run --select <MODEL> && data-diff --dbt
```
Comment (dlawin) [inline docs/os_diff/dbt_integration.md:188]: oh and these are missing the `--cloud`

Comment (williebsweet) [reply docs/os_diff/dbt_integration.md:188]: Good catch - I'll add

Comment (williebsweet) [reply docs/os_diff/dbt_integration.md]: https://github.com/datafold/datafold-docs/pull/136/commits/2ad59c0902daad2cc67c4931b4611274e8177367

File: docs/os_diff/dbt_integration.md (hunk 3 above)
Comment (williebsweet) [reply docs/os_diff/dbt_integration.md]: Done - https://github.com/datafold/datafold-docs/pull/136/commits/51cdb0be9db9d03a74efc474008f04cea7daaa13

**Review by williebsweet — COMMENTED** [review]
Approving - Dan's section

**Review by dlawin — APPROVED** [review]

**Pull request merged**